src/Microsoft.OData.Edm/Vocabularies/CommunityVocabularyConstants.cs

```diff
@@ -1,5 +1,5 @@
 ﻿//---------------------------------------------------------------------
-// <copyright file="AlternateKeysVocabularyConstants.cs" company="Microsoft">

```

Comment:
Maybe it's better to "[Obsolete]" for the existing "AlternateKeys*"  classes and files. 

Meantime, to create new for "Community*" classes and files?